.pylintrc

```diff
@@ -0,0 +1,2 @@
+[Format]
+max-line-length=80
```

Comment:
We were on the same page, I just wanted to make it explicit to anyone listening in. I'll push a commit in a moment that just prints the pylint output when the score is less than 9.25. I think anything above that mark can be safely considered well-formatted.